src/rev_ai/streamingclient.py

```diff
@@ -73,6 +73,9 @@ def start(self, generator, metadata=None):
             + '&content_type={}'.format(self.config.get_content_type_string()) \
             + '&user_agent={}'.format(quote('RevAi-PythonSDK/{}'.format(__version__), safe=''))
 
+        if custom_vocabulary_id:
```

Comment:
urlencode is the right way to create a URL. See https://github.com/revdotcom/revai-python-sdk/pull/32/files